build.sbt

```diff
@@ -58,17 +61,17 @@ lazy val `zio-telemetry` =
 
 testFrameworks := Seq(new TestFramework("zio.test.sbt.ZTestFramework"))
 
-val circeVersion  = "0.12.3"
```

Comment:
Can we move the version definitions together?

Reply:
Good point, done.